BonusCalcListener.Tests/Gateway/TimesheetGatewayTests.cs

```diff
@@ -41,118 +41,14 @@ public async Task RetrievesCorrectTimesheetIfClosedInOpenWeek()
         public async Task RetrievesCorrectTimesheetIfClosedInClosedWeek()
         {
             // Arrange
-            await SeedData();
+            await SeedData().ConfigureAwait(false);
```

Comment:
I see data is now seeded in DatabaseTests.cs, why do we still seed data in this test, unlike the test above?

Reply:
Fixed